ansible/roles/rt/tasks/main.yml

```diff
@@ -1,38 +1,175 @@
+    - { url: 'https://github.com/bestpractical/rt-extension-repeatticket/archive/master.zip', name: 'rt-extension-repeatticket' }
+    - { url: 'https://github.com/gitpan/RT-Extension-ResetPassword/archive/master.zip', name: 'rt-extension-resetpassword' }
+    - { url: 'https://github.com/bestpractical/rt-extension-spawnlinkedticketinqueue/archive/master.zip', name: 'rt-extension-spawnlinkedticketinqueue' }
+  when: rt_exists.stat.exists == false
+
+- name: extract all nagios plugins
+  unarchive: src=/opt/{{ item }}/master.zip dest=/opt/ copy=no
+  with_items:
+    - RT-Extension-ActivityReports
+    - rt-extension-commandbymail
+    - rt-extension-nagios
+    - rt-extension-mergeusers
+    - rt-extension-repeatticket
+    - rt-extension-resetpassword
+    - rt-extension-spawnlinkedticketinqueue
+  when: rt_exists.stat.exists == false
+
+
+# Due to the way the files get extracted, everything ends up in lower-case with 
+# -master, hence the item names below.
+
+- name: run perl Makefile.PL for all nagios plugins
+  command: perl Makefile.PL
+  args:
+    chdir: /opt/{{ item }}
+  with_items:
+    - rt-extension-activityreports-master
+    - rt-extension-commandbymail-master
+    - rt-extension-nagios-master
+    - rt-extension-mergeusers-master
+    - rt-extension-repeatticket-master
+    - RT-Extension-ResetPassword-master
+    - rt-extension-spawnlinkedticketinqueue-master
+  when: rt_exists.stat.exists == false
+
+- name: run make for nagios plugins
+  command: make
+  args:
+    chdir: /opt/{{ item }}
+  with_items:
+    - rt-extension-activityreports-master
+    - rt-extension-commandbymail-master
+    - rt-extension-nagios-master
+    - rt-extension-mergeusers-master
+    - rt-extension-repeatticket-master
+    - RT-Extension-ResetPassword-master
+    - rt-extension-spawnlinkedticketinqueue-master
+  when: rt_exists.stat.exists == false
+
+- name: run make install for nagios plugins
+  command: make install
+  args:
+    chdir: /opt/{{ item }}
+  with_items:
+    - rt-extension-activityreports-master
+    - rt-extension-commandbymail-master
+    - rt-extension-nagios-master
+    - rt-extension-mergeusers-master
+    - rt-extension-repeatticket-master
+    - RT-Extension-ResetPassword-master
+    - rt-extension-spawnlinkedticketinqueue-master
+  when: rt_exists.stat.exists == false
+
+
+
+# We already have a DB, so no need to initialise the database
+
+# Configure RT
+
+- name: configure rt server
+  template: src=RT_SiteConfig.pm dest=/opt/rt4/etc owner=root group=www-data mode=0660
+  when: rt_exists.stat.exists == false
+
+# Configure Procmail
 
 - name: configure procmail
-  copy: src=procmailrc.rt dest=/data/rt/rt_postfix/procmailrc.rt owner=root group=root mode=0644
+  copy: src=procmailrc.rt dest=/opt/rt4/etc/procmailrc.rt owner=root group=root mode=0644
+  when: rt_exists.stat.exists == false
+
+- name: Create .installed file now that RT is installed
+  file: path=/opt/rt4/.installed state=file
```

Comment:
This needs to be `state=touch`. `state=file` will NOT create a new file if it doesn't exist.